### DISCUSSION TITLE
Build Status: failed

### DISCUSSION THREAD

**Discussion by ehsan-askari** [codemagic-ci-cd/codemagic-docs#1914] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Build

### Expected results

Success Build

### Actual results

Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "IQKeyboardManagerSwift":
  In snapshot (Podfile.lock):
    IQKeyboardManagerSwift (= 6.5.12)

  In Podfile:
    IQKeyboardManagerSwift

None of your spec sources contain a spec satisfying the dependencies: `IQKeyboardManagerSwift, IQKeyboardManagerSwift (= 6.5.12)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Build failed :|
Step 3 script `Install CocoaPods dependencies` exited with status code 31

### Build id (optional)

64c0fe01dd894279d5c66683

**Comment by kulakarla** [comment, marked as answer]
Hello @ehsan-askari 

Try updating pod source repos as the error suggests.
Modify the script to be like this:

```
- name: Install CocoaPods dependencies
        script: | 
          pod repo update
          pod install
```

**Reply by ehsan-askari** [reply]
Fixed, Thanks a lot.
